Sources/Plot/API/HTMLComponents.swift

```diff
@@ -87,8 +87,12 @@ public extension Node where Context == HTML.HeadContext {
     ///   to the device the page is being rendered on. See `HTMLViewportWidthMode`.
     /// - parameter initialScale: The initial scale that the page should use.
     static func viewport(_ widthMode: HTMLViewportWidthMode,
-                         initialScale: Double = 1) -> Node {
-        let content = "width=\(widthMode.string), initial-scale=\(initialScale)"
+                         initialScale: Double = 1,
+                         fit: HTMLViewportFitMode? = nil) -> Node {
```

Comment:
One more quick thing (sorry!): Could you also document this new `fit` parameter?